core/sail/spin/src/test/java/org/eclipse/rdf4j/sail/spin/SpifSailTest.java

```diff
@@ -47,6 +48,8 @@
 
 	private RepositoryConnection conn;
 
+	private Locale locale;
```

Comment:
If it is just named "locale" it looks like a test variable rather than just a storage position for something unrelated to the test itself. Naming this "oldLocale" or "previousLocale" may be easier to remember its purpose in future, but otherwise it all looks good.